.gitignore

```diff
@@ -237,3 +237,6 @@ xcode
 *.nupkg
 *.exe
 bin
+
+#vcpkg
```

Comment:
We have VCPKG as a submodule so this needs to be here. If VCPKG is external, then it installs them in vcpkg_installed so we can have both. Ideally we should use the internal one though as I don't think the overlays will be applied right?

Reply:
I think this can be removed.  The `vcpkg_installed` directory is being stored in the build directory and that is already being ignored.